src/ex1/script.js

```diff
@@ -0,0 +1,165 @@
+const allTodosList = document.getElementById("all-todos-list");
+
+const unsorted = 0;
+const sortedAsc = 1;
+const sortedDesc = 2;
+let isListSorted = unsorted;
+
+const todosArray = ['Wash the dishes', 'Walk the dog', 'Water the flower', 'Feed the baby'];
+const inputTitle = document.getElementById("new-todo-title");
+const amountOfTodosInfo = document.getElementById("amount-info");
+
+const clearAllButton = document.getElementById("clear-all-button");
+const addTodoForm = document.getElementById("add-todo");
+const sortListButton = document.getElementById("sort-list-button");
```

Comment:
having all the elements outside here requires all the dom elements exist when this file is loaded, which might not be the case in complex web apps that dynamically change the dom depending on the view

Reply:
I think it is solved in Ex2 with document.addEventListener("DOMContentLoaded", ... )